frontend/src/metabase/collections/containers/CollectionSidebar.jsx

```diff
@@ -126,6 +128,22 @@ class CollectionSidebar extends React.Component {
             {t`View archive`}
           </Link>
         </Box>
+      </React.Fragment>
+    );
+  };
+
+  render() {
+    const { loaded } = this.props;
+    return (
+      <Sidebar w={340} pt={3} data-testid="sidebar">
+        {loaded ? (
+          this.renderContent()
+        ) : (
+          <div className="text-brand text-centered">
+            <LoadingSpinner />
+            <h2 className="text-normal text-light mt1">{t`Loading ...`}</h2>
```

Comment:
@kulyk we already have a translated string with an ellipsis so I would suggest using it instead of a new one with 3 dots: 
`Loading…`

Reply:
Good call! Honestly I just copy pasted it from the [`LoadingAndErrorWrapper`](https://github.com/metabase/metabase/blob/785060d44a1c4f1e2c547e432e8858969c9b9c05/frontend/src/metabase/components/LoadingAndErrorWrapper.jsx#L35). Makes sense to replace it there as weel

Reply:
Avoid making translation changes in dot-releases (eg 39.x), since we don't update translations until major releases.

Reply:
@flamber I guess we're safe here? If we're sure `Loading…` is covered with translations

Reply:
You'll have to ask someone who understand how translations are hooked into the frontend.
I can see there's three styles for this: `` t`Loading ...` `` and `` t`Loading...` `` and then a three-dot `` t`Loading…` ``